lighthouse-core/report/html/report-styles.css

```diff
@@ -260,6 +261,10 @@
   --audit-group-indent: 16px;
   --audit-indent: 16px;
   --expandable-indent: 16px;
+  --gauge-circle-size-big: 96px;
+  --gauge-circle-size: 72px;
+  /* Consider offset of the DevTools UI. */
```

Comment:
these feels slightly magical is there anyway we can link to how to compute this number in the future? :)

Reply:
So I actually just increased the offset value until the topbar was 100% visible and there was no gap between the DevTools UI + the topbar.

Think I should look into computing the offset at runtime, and somehow injecting the CSS variable?

Reply:
oh haha, nah maybe just make a note of "this is the height of the devtools UI as eyeballed by a developer in styles panel" or something? 😆